HuntHelper/Managers/IpcSystem.cs

```diff
@@ -13,35 +14,46 @@ namespace HuntHelper.Managers;
 public class IpcSystem : IDisposable
 {
     private const uint HuntHelperApiVersion = 1;
+    private const string IpiFuncNameEnable = "HH.Enable";
+    private const string IpcFuncNameDisable = "HH.Disable";
+    private const string IpcFuncNameGetVersion = "HH.GetVersion";
+    private const string IpcFuncNameGetTrainList = "HH.GetTrainList";
+    private const string IpcFuncNameImportTrainList = "HH.ImportTrainList";
```

Comment:
i put all the function names into these constants so there is one convenient place to look, in order to see all functions provided by hunt helper.